spec/commands/scrape_command_spec.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+SearchServiceError = GoogleScraperRuby::Errors::SearchServiceError
+
+RSpec.describe ScrapeCommand, type: :service do
+  describe '#run' do
+    context 'given an existing keyword_id' do
+      it 'updates the scrape result to the database if the command succeeds', vcr: 'google/google_no_ads' do
```

Comment:
Prefern waraping conditional into a context to adoid lengthy example descriptions:

```ruby
context 'given an existing keyword' do
  context 'given NO errors' do
    it 'save the scraped result', vcr: 'google/google_no_ads' do

    end
  end

  context 'given errors' do
    it 'raises an error' do

    end

    it 'logs the error' do

    end
  end
end
```

> **Note**
> Notice the capitalization of `NO` to make it easy at first glance to know what the test is about.

Reply:
Fixed in https://github.com/Goose97/google-scraper-ruby/pull/20/commits/9988cc8d24c0bff170a67f8cc38e7b31a6adf90d#diff-a64e6d1a295f3b99057c1fa3f636d0b6ce962ebb7c5957bbda89e526ab28795b